model/ftn/ww3_ounf.ftn

```diff
@@ -861,7 +862,11 @@
       NCID = 0
 !
 !
-      CALL U2D('days since 1990-01-01 00:00:00',REFDATE,IERR)
+      IF (CAL360) THEN
+        CALL U2D('days since 1900-01-01 00:00:00',REFDATE,IERR)
+      ELSE
+        CALL U2D('days since 1990-01-01 00:00:00',REFDATE,IERR)
+      ENDIF
```

Comment:
why do you need to set the reference data to 1900 if cal360 ?

Reply:
The 360-day calendar is usually employed in climatological studies, which run for several decades. If somebody wants to run a climatological study for several decades up to the present day (something that is done at the Met Office, for example), the start of the run is probably before the 1990 reference date. Of course we will have the same problem if we want to start the run before year 1900, so the choice of year is something that can be discussed. At least 1900 will allow running the most common climate jobs.

Reply:
it has no impact, the value will be negative if it's before 1990. I know that some codes have still the reference date hardcoded as 1990 so it should be better to keep 1990 in all cases.

Reply:
In that case I will just leave 1990 as the reference date. Thanks.

Reply:
Change looks OK to me

Reply:
Looks good to me.